src/buildercore/core.py

```diff
@@ -287,6 +309,7 @@ def stack_all_ec2_nodes(stackname, workfn, username=config.DEPLOY_USER, concurre
         workfn, work_kwargs = workfn
 
     data = stack_data(stackname)
+    # TODO: reuse all_node_params?
```

Comment:
perhaps `all_node_params` should have reused this? or this should have been extended to accommodate whatever usecase `all_node_params` needs. This is how a codebase grows gnarly.